tests/publiccloud/xfsprepare.pm

```diff
@@ -39,8 +39,11 @@ sub install_xfstests {
     ensure_user_exists("daemon", 2);
     # Create test users (See https://git.kernel.org/pub/scm/fs/xfs/xfstests-dev.git/tree/README)
     assert_script_run("useradd -mU fsgqa");    # Create home directory (-m) and 'fsgqa' group for the user (-U) as well
-    script_run("useradd 123456-fsgqa");    # script_run because only required by few tests and there is a chance that users starting with digits won't work
     assert_script_run("useradd fsgqa2");
+
+    # The following is only required by few tests and there is a chance that users starting with digits won't work
+    my $fsgqa_123456 = script_run("useradd 123456-fsgqa") == 0;
+    record_info("123456-fsgqa", "error creating 123456-fsgqa user.\nSome tests will not be able to run") unless ($fsgqa_123456);
```

Comment:
```suggestion
    record_info("123456-fsgqa", "error creating 123456-fsgqa user.\nSome tests will not be able to run", result => 'softfail') unless ($fsgqa_123456);
```

Reply:
I don't know, but since this will always fail and it is kinda known, I'm against putting a softfailure there. And yes, I know that this is the perfect situation FOR a softfailure.

The reason I'm against, is that there is nothing we gain from this softfailure. This issue will probably never being solved, and it is a known weekness of the xfstests, that is even present in the xfstests README.

WDYT?

Reply:
First of all I already approved this PR , so this is not something on which I really insists so it is up to you to take it or ignore it. 
Nevertheless : 
1. AFAIK `reuslt => 'softfail'`  is not the same as calling `record_softfailure` . While the last one will gives you yellow bubble and yellow module the first will just draw yellow square . And I find this good trade off between just "normal" record_info and actually doing softfail with `record_softfailure`. 
2. Isn't soft failed state is exactly about such situation ?  When you have some problem which have no clear future ?
3. You stating that "some tests will not be able" can you show me how this actually look like ? Is there in further failures or tests are just silently skipped ? 
4. If it is so common that you don't want to draw extra attention to it maybe we can remove this record_info ?

Reply:
1. That is nice, now I want to try it out :slightly_smiling_face: 
2. Yes but I also find it kinda .... disappointing :wink: 
3. It's actually only one that I could find: [generic/381](https://openqa.suse.de/tests/7811616#step/1_test_result/81) and yes, it's skipped.
4. Same as two: softfailures for something that is predictably never being solved is kinda unsettling. This is however only my personal taste and can differ from person to person

Gonna try your approach, if this does not alter the overall test result I find it a very good suggestion!

Reply:
http://duck-norris.qam.suse.de/tests/7941#step/xfsprepare/63 - I love this suggestion, this is amazing :+1: :+1: :+1: